selfdrive/boardd/boardd.cc

```diff
@@ -107,6 +107,8 @@ void safety_setter_thread() {
   cereal::CarParams::Reader car_params = cmsg.getRoot<cereal::CarParams>();
   cereal::CarParams::SafetyModel safety_model = car_params.getSafetyModel();
 
+  panda->set_unsafe_mode(0);  // see safety_declarations.h for allowed values
```

Comment:
Do you want it to send a zero by default or just provide the function and remove this?

Reply:
yeah, this is what we want